### PR TITLE
fix summary details

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx [hunk 1]
```diff
@@ -10,21 +12,40 @@ import { ORDER_BY_TYPE_PAGE_SIZE } from "../consts";
...
+    pex_pa_max: t("examination.ap.max"),
+    pex_pa_min: t("examination.ap.min"),
+    pex_fc: t("examination.heartrate"),
+    pex_temp: t("examination.temperature"),
+    pex_sat: t("examination.saturation"),
+    pex_note: t("examination.note"),
+    exam: t("lab.exam"),
+    material: t("lab.material"),
+    result: t("lab.result"),
+  };
```
Comment (almeida1492) [inline src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx:47]: We can use a function or even a custom hook to generate this object, given that this is the same for `PatientSummaryByType` and `PatientSummaryByDate`.

**Review by almeida1492 — CHANGES_REQUESTED** [review]

**Review by almeida1492 — APPROVED** [review]

File: src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx (hunk 1 above)
Comment (axelpezzo) [reply src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx]: @tsognong do you have some news about this?

Comment (tsognong) [reply src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx]: No.

Comment (axelpezzo) [reply src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx]: Nope just to be sure before merge this branch

**Pull request merged**